packages/facility-server/app/routes/apiv1/patient/patientProgramRegistration/patientProgramRegistrationConditions.js

```diff
@@ -1,54 +1,22 @@
 import express from 'express';
 import asyncHandler from 'express-async-handler';
 import { subject } from '@casl/ability';
-import { NotFoundError, ValidationError } from '@tamanu/shared/errors';
-import { validatePatientProgramRegistrationRequest } from './utils';
+import { NotFoundError } from '@tamanu/shared/errors';
 
 export const patientProgramRegistrationConditions = express.Router();
 
-patientProgramRegistrationConditions.post(
```

Comment:
Adding a condition directly is no longer a feature as conditions are added through the create or update program registry modal